packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -149,7 +149,6 @@ class SingleImageViewerContainer extends React.Component {
   async preload () {
     const { subject } = this.props
     if (subject && subject.locations) {
-      // TODO: Validate for allowed image media mime types
       const imageUrl = Object.values(subject.locations[0])[0]
```

Comment:
Is there a reason my TODO comment is being removed?

Reply:
🤦‍♂ I thought it was an IE related comment I removed, relevant before an earlier rebase, apologies